tests/unit/executor/test_part_handler.py

```diff
@@ -123,7 +123,7 @@ def test_run_build(self, mocker):
                 "installed-packages": ["hello=2.10"],
                 "installed-snaps": ["snapcraft=6466"],
             },
-            overlay_hash="6554e32fa718d54160d0511b36f81458e4cb2357",
+            overlay_hash="a3700a380077e2a4306b64628707ad87588040f2",
```

Comment:
All layer hashes changed because we added overlay-organize content to the digest.